sound/soc/intel/common/soc-acpi-intel-cml-match.c

```diff
@@ -59,42 +59,95 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cml_machines[] = {
 };
 EXPORT_SYMBOL_GPL(snd_soc_acpi_intel_cml_machines);
 
-static const u64 rt711_0_adr[] = {
-	0x000010025D071100
+static const struct snd_soc_acpi_endpoint single_endpoint = {
+	.num = 0,
+	.aggregated = 0,
+	.group_position = 0,
+	.group_id = 0,
```

Comment:
@plbossart do you think it makes sense to have a macro for this?

Reply:
For now it's not really useful, we have only 3 definitions. We can always add macros later if needed.